python/chilldkg_ref/vss.py

```diff
@@ -74,6 +74,32 @@ def commitment_to_secret(self) -> GE:
     def commitment_to_nonconst_terms(self) -> List[GE]:
         return self.ges[1 : self.t()]
 
+    def invalid_taproot_commit(self) -> Tuple[VSSCommitment, Scalar]:
+        # Create VSS commitment that Taproot-commits to a message that is
+        # invalid for BIP 341 script path spends.
```

Comment:
```suggestion
        # Return a modified VSS commitment whose corresponding  Taproot (BIP341) output key has an unspendable script path
```

It's hard to write a concise description here. I think this is the best I can come up with

Reply:
What exactly do you think is the problem with the current description? Happy for suggestions, but "corresponding Taproot output" sounds vague compared to the current wording. In particular, it does not make clear that the function creates a Taproot commitment.

Reply:
All fair points. I am using a variant of your last suggestion now. Wdyt?

Reply:
looks great